index.html

```diff
@@ -1,314 +1,259 @@
-              <i class="fab fa-twitter"></i>
-            </a>
-          </li>
-          <li class="list-inline-item social-facebook">
-            <a href="#">
-              <i class="fab fa-facebook-f"></i>
-            </a>
-          </li>
-          <li class="list-inline-item social-google-plus">
-            <a href="#">
-              <i class="fab fa-google-plus-g"></i>
-            </a>
-          </li>
-        </ul>
+        <h2>Make your own AR experience.<br />Register today!</h2>
+        <a href="http://dashboard.solsticear.com/register" class="btn btn-outline btn-xl js-scroll-trigger">Register
+          here</a>
       </div>
-    </section>
+    </div>
+    <div class="overlay"></div>
+  </section>
 
-    <footer>
-      <div class="container">
-        <p>&copy; Solstice AR.</p>
-        <ul class="list-inline">
-          <li class="list-inline-item">
-            <a href="privacy.html">Privacy Policy</a>
-          </li>
-          <!-- <li class="list-inline-item">
+  <section class="contact bg-primary" id="contact">
+    <div class="container">
+      <h2>
+        We
+        <i class="fas fa-heart"></i>
+        AR
+      </h2>
+      <ul class="list-inline list-social">
+        <li class="list-inline-item social-twitter">
+          <a href="#">
+            <i class="fab fa-twitter"></i>
+          </a>
+        </li>
+        <li class="list-inline-item social-facebook">
+          <a href="#">
+            <i class="fab fa-facebook-f"></i>
+          </a>
+        </li>
+        <li class="list-inline-item social-google-plus">
+          <a href="#">
+            <i class="fab fa-google-plus-g"></i>
+          </a>
+        </li>
+      </ul>
+    </div>
+  </section>
+
+  <footer>
+    <div class="container">
+      <p>&copy; Solstice AR.</p>
+      <ul class="list-inline">
+        <li class="list-inline-item">
+          <a href="privacy.html">Privacy Policy</a>
+        </li>
+        <!-- <li class="list-inline-item">
             <a href="#">Terms</a>
           </li>
           <li class="list-inline-item">
             <a href="#">FAQ</a>
           </li> -->
-        </ul>
-      </div>
-    </footer>
+      </ul>
+    </div>
+  </footer>
+
+  <!-- Bootstrap core JavaScript -->
+  <script src="vendor/jquery/jquery.min.js"></script>
+  <script src="vendor/bootstrap/js/bootstrap.bundle.min.js"></script>
 
-    <!-- Bootstrap core JavaScript -->
-    <script src="vendor/jquery/jquery.min.js"></script>
-    <script src="vendor/bootstrap/js/bootstrap.bundle.min.js"></script>
+  <!-- Plugin JavaScript -->
+  <script src="vendor/jquery-easing/jquery.easing.min.js"></script>
 
-    <!-- Plugin JavaScript -->
-    <script src="vendor/jquery-easing/jquery.easing.min.js"></script>
+  <!-- Custom scripts for this template -->
+  <script src="js/new-age.min.js"></script>
+</body>
 
-    <!-- Custom scripts for this template -->
-    <script src="js/new-age.min.js"></script>
-  </body>
-</html>
+</html>
```

Comment:
add line

Reply:
Should only add new lines for JS files that need to be minified.